alerting/src/test/kotlin/org/opensearch/alerting/MonitorDataSourcesIT.kt

```diff
@@ -341,6 +342,47 @@ class MonitorDataSourcesIT : AlertingSingleNodeTestCase() {
         assertEquals("Didn't match all 4 queries", 1, findings[0].docLevelQueries.size)
     }
 
+    fun `test execute monitor without create when no monitors exists`() {
```

Comment:
Lets have one more test where the monitor is created and dryrun is true and there was not update to the metadata doc

Reply:
Not actually valid to have a test like that.